### ISSUE TITLE
Change the command line...

### ISSUE THREAD

**Issue by MaiklT** [DNNCommunity/DNN.Newsletter#9]
The command line mentioned in the release 8.0.0 could fail if the database owner is not dbo and/or an object qualifier is used

Therefore it should be

`DELETE FROM {databaseOwner}{objectQualifier}ModuleDefinitions where Friendlyname='Newsletters'`

Anyway, I would include a 08.00.00.SqlDataProvider file containing

```
DECLARE @Count int

SELECT @Count = COUNT(*) FROM {databaseOwner}{objectQualifier}ModuleDefinitions WHERE Friendlyname = 'Newsletters'

IF @Count > 0
BEGIN
   DELETE FROM {databaseOwner}{objectQualifier}ModuleDefinitions WHERE Friendlyname = 'Newsletters'
END
```

**Comment by david-poindexter** [comment]
@MaiklT I can confirm that this is not necessary in a fresh install of DNN 09.01.01.  However, I concur with your recommendations.  Your recommendation checks to see if the record exists first and them removes it accordingly if it does exist.  I am going to see if I can get this project to build locally and I will include if I am successful.  Thanks!

**Comment by david-poindexter** [comment]
Closing this issue manually, as #13 has now been merged into the `release/08.00.01` branch and will be merged into the `master` branch for the 08.00.01 release.
